corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -46,8 +46,8 @@
                   "><i class="fa fa-remove"></i></button>
         </legend>
       </div>
-      <div data-bind="if: collapsible">
-        <div class="collapsible-icon-container">
+      <div data-bind="if: collapsible" style="display: flex">
+        <div class="collapsible-icon-container" style="align-self: center">
```

Comment:
Does Bootstrap's `center-block` class work here instead of `align-self: center`?